src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -98,31 +137,136 @@ export default {
 			return this.virtualBackground.isAvailable()
 		},
 
-		backgrounds() {
-			return [
-				imagePath('spreed', 'backgrounds/1.jpg'),
-				imagePath('spreed', 'backgrounds/2.jpg'),
-				imagePath('spreed', 'backgrounds/3.jpg'),
-				imagePath('spreed', 'backgrounds/4.jpg'),
-				imagePath('spreed', 'backgrounds/5.jpg'),
-				imagePath('spreed', 'backgrounds/6.jpg'),
-			]
+		isCustomBackground() {
+			return !this.predefinedBackgroundsURLs.includes(this.selectedBackground)
+				&& this.selectedBackground !== 'none'
+				&& this.selectedBackground !== 'blur'
```

Comment:
```suggestion
			return this.selectedBackground !== 'none
			    && this.selectedBackground !== 'blur'
			    && !this.predefinedBackgroundsURLs.includes(this.selectedBackground)
```

I think it's better to check for string properties first, rather than look through an array